### ISSUE TITLE
Impaired neuronal physiology 

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7042]
**Preferred term label:**

Impaired neuronal physiology 

**Synonyms**

Neuronal dysfunction

**Definition (free text, please give PubMed ID)**

Any functional anomaly resulting in the reduced function of a neuron.
PMID:9130294

**Parent term (use hpo.jax.org/app)**

abnormal neuron physiology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

Alzheimer's Disease (OMIM: 104300)

**Your nano-attribution (ORCID)**

https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
I am not convinced this is a good HPO for the reasons stated here
https://github.com/obophenotype/human-phenotype-ontology/issues/7055
If there is a specific measurement of neuronal physiology that is used in a clinical or translational research context then please reopen this issue. The HPO does not intend to be an ontology of abnormal physiology, and so the (correct) inference that there is abnormal neuronal physiology is not a valid HPO term unless there is a concrete measurement that we can point to for a specific kind of abnormality.
